projects/igniteui-angular/src/lib/carousel/carousel.component.ts

```diff
@@ -535,9 +537,13 @@ export class IgxCarouselComponent extends IgxCarouselComponentBase implements On
         this.restartInterval();
     }
 
-    constructor(cdr: ChangeDetectorRef, private element: ElementRef, private iterableDiffers: IterableDiffers,
-        builder: AnimationBuilder, private platformUtil: PlatformUtil) {
-        super(builder, cdr);
+    constructor(
+        cdr: ChangeDetectorRef,
```

Comment:
```suggestion
        private cdr: ChangeDetectorRef,
```

Reply:
this is already marked as private in base class